crates/aptos-crypto/src/secp256r1_ecdsa/secp256r1_ecdsa_sigs.rs

```diff
@@ -71,7 +71,7 @@ impl Signature {
         Ok(())
     }
 
-    /// Check if S < ORDER_HALF to capture invalid signatures.
+    /// Check if S <= ORDER_HALF to capture invalid signatures.
```

Comment:
Can you rename this to `check_s_lt_order_half`?

Reply:
I have renamed the function to `check_s_le_order_half` as requested. The changes have been committed under hash `f46e76c21551f17ca7b0a04bd473b50e7d6c11e8`.